code/modules/admin/admin_verbs.dm

```diff
@@ -781,6 +783,81 @@ var/list/admin_verbs_mod = list(
+	//remains here to remind you of what could have been.
+	/*
+	var/obj/item/projectile/tempproj = new
+	var/list/projectile_vars = sortList(tempproj.vars)
+	qdel(tempproj)
+	*/
+
+	var/list/projectile_vars = list(
+		//damage/main vars
+		"damage", "damage_type", "nodamage", "what armor resists (flag)", "projectile_speed", "travel_range",
+		//effect vars
+		"stun", "weaken", "paralyze", "irradiate", "eyeblur", "drowsy", "agony", "jittery",
+		//appearance/misc vars
+		"icon", "icon_state", "color", "silenced", "manual variable entry"
+	)
+
+	var/inputvar
+	if(!G.bullet_overrides)
+		G.bullet_overrides = list()
+	if(!G.bullet_overrides.len)
+		inputvar = input(usr, "What variable would you like to change?", "Gun Variables") as null|anything in projectile_vars
+	else
+		var/existing_bullet_override_pick = input(usr, "Would you like to edit an existing variable or add a new one?", "Gun Variables") as null|anything in G.bullet_overrides + "(ADD VAR)"
+		if(!existing_bullet_override_pick)
+			return
+		else if(existing_bullet_override_pick == "(ADD VAR)")
+			inputvar = input(usr, "What variable would you like to change?", "Gun Variables") as null|anything in projectile_vars
+		else
+			inputvar = existing_bullet_override_pick
+	if(!inputvar)
+		return
+	var/newvalue
+	switch(inputvar)
+		if("damage","irradiate")
+			newvalue = input(usr, "How much damage should the projectile deal?", "Var: [inputvar]") as null|num
+		if("damage_type")
+			newvalue = input(usr, "What type of damage should the projectile deal (it can only deal one, sorry)?", "Var: [inputvar]") as null|anything in list("brute", "oxy", "tox", "fire", "clone", "brain")
+		if("nodamage")
+			newvalue = input(usr, "Should the gun do damage on hit?", "Var: [inputvar]") as null|anything in list(TRUE, FALSE)
+		if("what armor resists (flag)")
+			newvalue = input(usr, "What armor type should resist this damage?", "Var: [inputvar]") as null|anything in list("melee", "bullet", "laser", "energy", "bomb", "bio", "rad")
+			inputvar = "flag"
+		if("projectile_speed")
+			newvalue = input(usr, "How much time (in deciseconds) should the bullet delay between tile movements? (examples, Taser electrode is 1, Glock bullets are 0.5)?", "Var: [inputvar]") as null|num
+		if("travel_range")
+			newvalue = input(usr, "How far should the projectile travel before vanishing? (0 to go indefinitely)", "Gun Variables") as null|num
+		if("stun","weaken","paralyze","eyeblur","agony","jittery")
+			newvalue = input(usr, "New effect value? Most go down 1 per tick.", "Var: [inputvar]") as null|num
+		if("color")
+			newvalue = input(usr, "What color (RGB format, example: #00ff00)?", "Var: [inputvar]") as null|color
+		if("silenced")
+			newvalue = input(usr, "Should the gun have no text when fired?", "Var: [inputvar]") as null|anything in list(TRUE, FALSE)
+		if("manual variable entry")
+			inputvar = input(usr, "What variable name to manually insert and then edit (verify it!)?", "Var: [inputvar]") as null|text
```

Comment:
Should check if the inputted var exists after this to prevent unforeseen consequences.

Reply:
Cannot at this point in the code, but thankfully the bullet_overrides check later does. It only adds variables that already exist in the fired projectile.